apps/user/profile/index.coffee

```diff
@@ -55,7 +58,20 @@ class ProfileView extends Backbone.View
       view.initBlockViews()
       
 module.exports.init = ->
+  user = new User USER
+
   view = new ProfileView
     el: $('.profile')
 
-  view.setUpChannelGroupViews(PROFILE_CHANNELS)
\ No newline at end of file
+  view.setUpChannelGroupViews(PROFILE_CHANNELS)
+
+  new PathView
+    el: $('section.path--header')
+    model: user
+
+  new MetaEditableAttributeView
+    model: user
+    el: $("#metadata--about .metadata__content")
```

Comment:
From now on let's make it a point to only reference `js-` prefixed classes in javascript and to never style them.